projects.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: page
+title: Current Projects
+descriptiveTitle: Toronto Mesh Projects
+order: 1
+parent: Get Involved
+---
+
```

Comment:
Could there be preamble to the content of this page? Might be nice to tie in a prompt to suggest new a project if there isn't one of interest listed, along with something like a `Mesh 101` link, or resource, to get familiar with concepts, terms, etc.

Reply:
I added a blurb at the top of page- any suggestions for Mesh 101 link?

Reply:
For some reach I thought we had a dedicated domain/url for the Mesh 101 content. Ok with not mentioning it for brevity.